src/transformStateWithClones.js

```diff
@@ -8,6 +9,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const table = {
+    ...state,
+  };
+  const result = [];
+
+  for (const action of actions) {
+    if (action.type === 'clear') {
+      for (const delet in table) {
+        delete table[delet];
+      }
+      console.log(table);
```

Comment:
Avoid using `console.log` in the function body as it is meant for debugging and should not be part of the final implementation.